rename_images.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+files = os.listdir('./data/')
+
+image_info = {}
+for f in files:
+    parts = os.path.splitext(f)[0].split('_')
+    if len(parts) <= 2:
+        continue
+    webcompat_id = parts[0]
+    sequence_no = parts[-2]
+    elem_id = parts[1:-2]
+    browser = parts[-1]
+    if webcompat_id not in image_info.keys():
+        image_info[webcompat_id] = {}
+    image_info[webcompat_id][sequence_no] = elem_id
+    new_name = "%s_%s_%s.png" % (webcompat_id, sequence_no, browser)
+    os.rename("data/" + f, "data/" + new_name)
+
+for webcompat_id, seq_no_and_elem_id in image_info.items():
+    with open("./data/%s.txt" % webcompat_id, "w") as text_file:
+        seq_no_and_elem_id = sorted(seq_no_and_elem_id.items())
+        for value in seq_no_and_elem_id:
+            sequence_no = value[0]
+            elem_id = value[1][0]
```

Comment:
This is going to use only the first part of the element ID (e.g. for `an_element_id`, this variable would be `an`). I would replace line 12 with `'_'.join(parts[1:-2])` and this line with `value[1]`.

Reply:
@marco-c Done!